components/Dashboard/Tiles/EntryTile/EntryTile.js

```diff
@@ -16,46 +14,46 @@ export default function EntryTile({
   location,
 }) {
   const { data: session } = useSession();
-  const router = useRouter();
 
   return (
     <>
-      <Styled.Container>
+      <Styled.OuterContainer>
         <LinkWrapper link={`/id:${entryUrl}`}>
-          <Styled.TextContainer>
-            {session ? (
-              "You "
-            ) : (
-              <Styled.StaticText>Somebody </Styled.StaticText>
-            )}
-            {location === "unknown" ? (
-              ""
-            ) : (
-              <>
-                <Styled.StaticText>in </Styled.StaticText>
-                {location.region}
-              </>
-            )}
-            <Styled.StaticText> felt</Styled.StaticText> {experience}.{" "}
-            <Styled.StaticText>More specifically</Styled.StaticText>{" "}
-            <Intensity value={intensity} experience={experience} />
-            <Styled.StaticText>
-              . {session ? "You" : "They"} selected these tags:
-            </Styled.StaticText>{" "}
-            {reactions.map((reaction, index, array) => (
-              <span key={index}>
-                {reaction}
-                {index < array.length - 1 && ", "}
-              </span>
-            ))}
-          </Styled.TextContainer>
-          <div>
-            {getWeekdayFromTime(time)},<br />
-            {time}
-          </div>
-          <Styled.ColorCircle color={color} />
+          <Styled.Container>
+            <Styled.ColorCircle color={color} />
+            <Styled.TextContainer>
+              {session ? (
+                "You "
+              ) : (
+                <Styled.StaticText>Somebody </Styled.StaticText>
+              )}
+              {location === "unknown" ? (
```

Comment:
We could potentially simplify this by checking if there is a location.region property, otherwise show nothing.

i.e. 

``` js
{location?.region && (
  <Styled.StaticText>in {location.region}</Styled.StaticText>
)}
```